### PR TITLE
Repeat word

### PR DESCRIPTION
**Pull request by lhchandler4**
- [ X] Top-level README “Table of Contents” is updated
 - [ X] Feature tasks for this challenge are completed
 - [ X] Unit tests written and passing
     - [ X] “Happy Path” - Expected outcome
     - [ X] Expected failure
     - [ X] Edge Case (if applicable/obvious)
 - [ X] README for this challenge is complete
     - [ X] Summary, Description, Approach & Efficiency, Solution
     - [ X] Link to code
     - [ X] Picture of whiteboard

### CODE REVIEW

File: challenges/src/main/java/Java/RepeatWord/RepeatWord.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+import java.util.HashMap;
+
+public class RepeatWord {
+    public static String repeatWord(String input) {
+        String indivWord[] = input.split(" ");
+        String current = "";
+        HashMap<String, Integer> wordsHash = new HashMap<>();
+        for (String palabra: indivWord) {
+            current = palabra.toLowerCase().replaceAll("[^a-zA-Z]+$", "");
```
Comment (evanslaton) [inline challenges/src/main/java/Java/RepeatWord/RepeatWord.java:11]: I think `[^a-z]` covers everything you need, especially since you `.toLowerCase()` it all

File: challenges/src/main/java/Java/RepeatWord/RepeatWord.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+            if (wordsHash.containsKey(current)) {
+                return current;
+            } else if (input == "") {
```
Comment (evanslaton) [inline challenges/src/main/java/Java/RepeatWord/RepeatWord.java:14]: Remember to always use `.equals()` when comparing Strings

File: challenges/src/main/java/Java/RepeatWord/RepeatWord.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+                return ("Namaste");
```
Comment (evanslaton) [inline challenges/src/main/java/Java/RepeatWord/RepeatWord.java:15]: This might confuse users, is Namaste the first repeated word?
